rpcs3/Emu/System.cpp

```diff
@@ -1359,9 +1369,13 @@ game_boot_result Emulator::Load(const std::string& title_id, bool add_only, bool
 
 			const std::string game_path = "/dev_hdd0/game/" + m_path.substr(hdd0_game.size(), 9);
 
-			sys_log.notice("Forcing manual lib loading mode");
```

Comment:
Someone smoked somthing real good when deciding HLE all libs for PS1 classics... well fixed.

Reply:
I think it had its reasons at its time